apps/finance/contracts/Finance.sol

```diff
@@ -380,11 +383,11 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
     * @return success Boolean indicating whether the accounting period is the correct one (if false,
     *                 maxTransitions was surpased and another call is needed)
     */
-    function tryTransitionAccountingPeriod(uint64 _maxTransitions) public isInitialized returns (bool success) {
+    function tryTransitionAccountingPeriod(uint64 _maxTransitions) external isInitialized returns (bool success) {
```

Comment:
Given that `_tryTransitionAccountingPeriod` works with `uint256`, is there any reason to use `uint64` here?

Reply:
Will fix in another PR.